invenio_app_ils/ui/backoffice/src/index.css

```diff
@@ -0,0 +1,14 @@
+body {
```

Comment:
Overriding styles should come last, it would be good to use whatever the ui framework we use provides out of the box and customise it further down the path.

Reply:
same as above.